Fix pe-d bugs in user guide and developer guide (#267)
AY2425S1-CS2103T-T14-2/tp: docs/UserGuide.md

</div>

**Format 2**: `deletetag all t/TAG [t/MORE_TAG]…​`

| Parameter Name | Description                                                               | Required   |
|----------------|---------------------------------------------------------------------------|------------|
| `all`          | Indicates that the deletion operation applies to all contacts in the list | Compulsory |
| `t/TAG`        | Tag to be deleted from every contact in the list associated with the tag  | Compulsory |
| `t/MORE_TAG`   | More tags to be deleted                                                   | Optional   |

<div markdown="block" class="alert alert-info">

🔔 **Notes**: <br>

* `deletetag all t/TAG` deletes the tag `TAG` from all contacts currently shown in the list, not all contacts in the database.
* `deletetag all t/TAG` only works when all contacts currently shown in the list have the tag `TAG`, otherwise an error message is shown.
* For both formats 1 and 2, 
  * The tag is case-sensitive. For example, `partner` will not match `PartnEr`.
  * Only full words will be matched e.g. `partner` will not match `partners`.
</div>

<div markdown="span" class="alert alert-primary">💡
 **Tip**:

> To delete all contacts with the `Y3` tag, simply use `filtertag Y3` followed by `deletetag all t/Y3`

Reviewer: Perhaps rephrase this tip to sound more of deleting all tags than deleting all contacts to prevent misleading the user into thinking they will delete all contacts that have the Y3 tag

```suggestion
To delete all `Y3` tags from contacts, simply use `filtertag Y3` followed by `deletetag all t/Y3`
```

or another example regarding the student memberships like:
```suggestion
To delete all `paid` tags from students, simply use track student followed by `deletetag all t/paid`, when their memberships have expired and ready to be renewed 
```

Author: good suggestion, thanks
